packages/baseclient-components/src/container/Footer/Footer.tsx

```diff
@@ -22,13 +39,77 @@ interface FooterState {
  */
 export default class Footer extends React.Component<FooterProps, FooterState> {
 
+  private predefinedCrsDefinitions = [{
+    code: '4326',
+    value: 'WGS 84'
+  }, {
+    code: '3857',
+    value: 'Pseudo-Mercator -- Spherical Mercator'
+  }];
+
   /**
    * Create the Footer.
    *
    * @constructs Footer
    */
   constructor(props: FooterProps) {
     super(props);
+
+    this.setProjection = this.setProjection.bind(this);
+  }
+
+  /**
+   * Is invoked before a mounted component receives new props.
+   *
+   * @param {Object} nextProps The next props.
+   */
+  componentDidMount() {
+    const {
+      map
+    } = this.props;
+    if (map) {
+      this.createOlMousePositionControl(map);
+    }
+  }
```

Comment:
Even if it's very unlikely, that the Footer will be unmounted, but you might want to remove the `ol/control/MousePosition` from the map in this case.